src/main/java/ca/on/oicr/gsi/dimsum/service/filtering/CompletedGate.java

```diff
@@ -198,7 +178,7 @@ public Predicate<Sample> samplePredicate(MetricCategory requestCategory) {
   }
 
   public boolean qualifyCase(Case kase) {
-    return kase.getTests().stream().allMatch(test -> qualifyTest(test));
+    return kase.getTests().stream().anyMatch(test -> qualifyTest(test));
```

Comment:
This is changing it back to how it was. It should be left as `allMatch` so that a gate is only considered complete for the case if it is complete for all of the tests